### PR TITLE
[RDBMS] `az postgres flexible-server fabric-mirroring/identity`: Support system assigned managed identity and fabric mirroring of databases

### PR DESCRIPTION
**Pull request by nasc17**
**Related command**
`az postgres flexible-server identity`
`az postgres flexible-server fabric-mirroring`

**Description**
Add support to prepare and bring your PostgreSQL data into Microsoft Fabric.
Enable system assigned managed identity on the server by setting argument `--system-assigned Enabled`

Enable fabric mirroring by using `az postgres flexible-server fabric-mirroring start`
Disable fabric mirroring by using `az postgres flexible-server fabric-mirroring stop`
Update the databases being mirrored by using `az postgres flexible-server fabric-mirroring update-databases`

**Testing Guide**
Manually and test

**History Notes**
[RDBMS] `az postgres flexible-server fabric-mirroring` and `az postgres flexible-server identity`: Support system assigned managed identity and fabric mirroring of databases

### CODE REVIEW

**Comment by nasc17** [comment]
@evelyn-ys please trigger checks and review.

File: src/azure-cli/azure/cli/command_modules/rdbms/_helptext_pg.py [hunk 1]
```diff
@@ -1126,3 +1134,32 @@
...
+type: group
+short-summary: Bring your PostgreSQL data into Microsoft Fabric. Mirroring allows you to create a replica of your data in OneLake which can be used for all your analytical needs.
+"""
+
+helps['postgres flexible-server fabric-mirroring start'] = """
+type: command
+short-summary: Enable bringing your PostgreSQL data into Microsoft Fabric.
+examples:
+  - name: Enable bringing your PostgreSQL data into Microsoft Fabric.
+    text: az postgres flexible-server fabric-mirroring start -g testgroup -s testsvr --database-names testdb
```
Comment (mattboentoro) [inline src/azure-cli/azure/cli/command_modules/rdbms/_helptext_pg.py:1148]: nit: maybe you can share example where there are multiple database names

**Review by mattboentoro — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/rdbms/_helptext_pg.py (hunk 1 above)
Comment (nasc17) [reply src/azure-cli/azure/cli/command_modules/rdbms/_helptext_pg.py:1148]: We show that example in update-databases

**Comment by nasc17** [comment]
@wangzelin007 Hi could you help, I am unsure as to why the linter check is saying tests are missing:

_Missing command test coverage: `postgres flexible-server list-skus`_
Located [here](https://github.com/Azure/azure-cli/blob/616496b3f7f32ce233f2e1480fc734a70a84d95e/src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_flexible_commands.py#L220)
_Missing command test coverage: `postgres flexible-server show-connection-string`_
Located [here](https://github.com/Azure/azure-cli/blob/616496b3f7f32ce233f2e1480fc734a70a84d95e/src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_flexible_commands.py#L211)

**Comment by nasc17** [comment]
> @wangzelin007 Hi could you help, I am unsure as to why the linter check is saying tests are missing:
> 
> _Missing command test coverage: `postgres flexible-server list-skus`_ Located [here](https://github.com/Azure/azure-cli/blob/616496b3f7f32ce233f2e1480fc734a70a84d95e/src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_flexible_commands.py#L220) _Missing command test coverage: `postgres flexible-server show-connection-string`_ Located [here](https://github.com/Azure/azure-cli/blob/616496b3f7f32ce233f2e1480fc734a70a84d95e/src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_flexible_commands.py#L211)

@evelyn-ys any guidance here?

**Comment by nasc17** [comment]
@evelyn-ys I decided to redo test for the commands linter is saying are missing. Please trigger build and review. If same issue comes up I will follow the advice to just move back code placement.

**Comment by nasc17** [comment]
@evelyn-ys new tests did not stop failure for one of the commands. Reverted test results change and put command back in original location. Please rerun build, thank you for the support.

**Comment by evelyn-ys** [comment]
@nasc17 Please resolve conflict and I will trigger CI then

**Review by evelyn-ys — APPROVED** [review]

**Pull request merged**